### PR TITLE
Fix/swap transfer batch fix

### PR DESCRIPTION
**Pull request by pakrentos**
# Changes
- Added SwapBatchInfo struct, responsible for holding information about outcome asset, dex ID and related batch receivers
- Fixed run_script 
- Updated README.MD with instructions for macOS

### CODE REVIEW

**Comment by pakrentos** [comment]
I accidentally merged this branch with master. Is it ok or I should revert commits?

**Review by wer1st — APPROVED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -21,7 +21,7 @@ String contractsEnvFile       = 'env.template'
 String solcVersion            = '0.8.14'
 String nodeVersion            = '14.16.1'
 String gitHubUser             = 'sorabot'
-String gitHubRepo             = 'github.com/soramitsu/sora2-network.git'
+String gitHubRepo             = 'github.com/sora-xor/sora2-network.git'
```
Comment (wpi86) [inline Jenkinsfile:24]: you can get this from env variable

File: pallets/liquidity-proxy/src/lib.rs [hunk 2]
```diff
@@ -1524,6 +1524,30 @@ impl<T: Config> std::fmt::Debug for BatchReceiverInfo<T> {
...
+    pub receivers: Vec<BatchReceiverInfo<T>>,
+}
+
+impl<T: Config> SwapBatchInfo<T> {
+    pub fn len(&self) -> usize {
+        self.receivers.len()
+    }
+}
+
+#[cfg(feature = "std")]
```
Comment (vovac12) [inline pallets/liquidity-proxy/src/lib.rs:1541]: You can add `RuntimeDebug` derive macro to struct instead

File: run_script.sh [hunk 3]
```diff
@@ -83,7 +83,7 @@ for name in alice bob charlie dave eve ferdie
 do
 	newport=`expr $port + 1`
 	rpcport=`expr $wsport + 10`
-	$binary key insert --chain $chain --suri "//${name^}" --scheme ecdsa --key-type ethb --base-path db$num
+	$binary key insert --chain $chain --suri "//${name}" --scheme ecdsa --key-type ethb --base-path db$num
```
Comment (vovac12) [inline run_script.sh:86]: Why?)

**Comment by vovac12** [comment]
Please add this extrinsic to https://github.com/sora-xor/sora2-network/blob/e1258c1d13c4d790d9d7ea80f99679f33a1db6e8/runtime/src/extensions.rs#L112

File: run_script.sh (hunk 3 above)
Comment (pakrentos) [reply run_script.sh:86]: It produced "bad substitution" error on mac, though `echo "//${name^}"` worked as expected and produced names with first uppercase letter. 
I removed `^` and error disappeared, while testnet worked ok

**Pull request closed without merging**